ietf/doc/views_review.py

```diff
@@ -621,7 +621,7 @@ def clean_review_url(self):
         url = self.cleaned_data['review_url']
         #scheme, netloc, path, parameters, query, fragment = urlparse(url)
         if url:
-            r = requests.get(url)
+            r = requests.get(url, timeout=settings.DEFAULT_REQUESTS_TIMEOUT)
             if r.status_code != 200:
                 raise forms.ValidationError("Trying to retrieve the URL resulted in status code %s: %s.  Please provide an URL that can be retrieved." % (r.status_code, r.reason))
```

Comment:
The addition of the timeout opens the possibility of raising a `requests.exceptions.Timeout` exception.  It would be better to handle that.